docs/url-allow-list

```diff
@@ -66,3 +66,4 @@ ghe.example.com
 192.168.99.100:8443
 github.com/yourghuser/argo-cd
 github.com/argoproj/argo-cd/releases/download/
+github.com/argoproj/argo-helm/tree/master/charts/argo-cd
```

Comment:
Why is it necessary to allowlist this? It resolves fine for me, the the URL in installation.md matches this exactly.

Reply:
My bad. It failed on another URL in the file, which I hadn't noticed before.